CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+# Contributing to a Project Cognoma Repository
+
+## Project Philosophy
+
+Project Cognoma is an open source community-based project. Beyond our immediate scientific aspirations, we want to engage as many individuals as possible in doing cool work. Project Cognoma is a place to learn and grow as a community. So don't be shy. We want your contribution. Given the nature of open source development, not all contributions can be included … but all will be appreciated and help the project bloom.
+
+## Issues
+
+We use GitHub Issues for a wide range of tasks including bug reports, feature requests, planning, and general forum. Don't worry about whether your use of GitHub Issues is legitimate, but do try to pick the best [repository](https://github.com/cognoma "Repositories of the Cognoma Organization") to file your issue under.
+
+## Pull Requests
+
+Project Cognoma operates on a pull request model. Substantial changes should be submitted via [pull request](https://help.github.com/articles/using-pull-requests/ "GitHub · Using pull requests"). We also encourage you to comment on and review open pull requests.
```

Comment:
You may want to state "all" or "any" changes, not just substantial ones. Hot fixes or emergencies may need to bypass PRs, but that should be rare.